spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/metrics/SparkTelemetryEvents.java

```diff
@@ -0,0 +1,25 @@
+package com.google.cloud.spark.bigquery.metrics;
+
+import com.google.cloud.spark.events.BigQueryConnectorVersionEvent;
+import com.google.cloud.spark.events.InputFormatEvent;
+import java.io.Serializable;
+import org.apache.spark.sql.SQLContext;
+
+public class SparkTelemetryEvents implements Serializable {
```

Comment:
Consider adding serialVersionUID since the class is Serializable. Also for other classes.